modules/040-node-manager/hooks/handle_node_templates.go

```diff
@@ -193,6 +193,11 @@ func nodeTemplatesHandler(input *go_hook.HookInput) error {
 				}
 			}
 
+			// Prevent node deletion by autoscaler
+			if nodeObj.Labels["node.deckhouse.io/type"] == "CloudPermanent" || nodeObj.Labels["node.deckhouse.io/type"] == "Static" {
```

Comment:
```suggestion
			if set.New("CloudPermanent", "CloudStatic", "Static").Has(nodeObj.Labels["node.deckhouse.io/type"]) {
```

Little deckhouse magic

Reply:
done